yarn.lock

```diff
@@ -635,18 +635,21 @@
     framer-motion "^1.8.3"
     react-tooltip "^3.11.4"
 
-"@navikt/familie-form-elements@1.0.6-alpha.2+a8ced8e":
-  version "1.0.6-alpha.2"
-  resolved "https://registry.yarnpkg.com/@navikt/familie-form-elements/-/familie-form-elements-1.0.6-alpha.2.tgz#11e41b35938cfd994cb4154d8b5853791a64ee91"
-  integrity sha512-a9WzAMVdFQdIoR/Z1n+KcfAOH3B/RInvZXOxI+5QgHpU2gCklAwiOO830xXP85+B7RbujlLPgsWKa7EK5J2bng==
+"@navikt/familie-form-elements@1.0.7":
+  version "1.0.7"
+  resolved "https://registry.yarnpkg.com/@navikt/familie-form-elements/-/familie-form-elements-1.0.7.tgz#0c2828529be562b3dedd045cee7410f26c2c09ed"
+  integrity sha512-Ga++MtSSILy7bAt5KB8Bc5V159AY83dnfREm7dVg/++nYOEmnt88VJjjR/QflLggfh9t/LCGVxyziJY89sadCQ==
   dependencies:
     "@navikt/fnrvalidator" "^1.1.2"
+    nav-datovelger "^6.1.8"
```

Comment:
Denne skal opp på 6.1.9 for å få med siste ekstra støtte for datoformater, men hvis det ikke er noe mer som skal inn i familie-felles-frontend nå så oppdaterer jeg der og får hjelp av dere til å publishe :D